src/GroupResolverHandlerInterface.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\og;
+
+/**
+ * @file
+ * Contains \Drupal\og\GroupResolverHandlerInterface.
+ */
+
+/**
+ * Interface OgContextHandlerInterface.
+ *
+ * @package Drupal\og
+ */
+interface GroupResolverHandlerInterface {
```

Comment:
What is the purpose of this class? The documentation doesn't explain it. Why do we need an additional class to handle GroupResolver plugins? Isn't this the task of `GroupResolverManager`?

This is confusing to me since this class is only used in tests so far, there is no real code calling into it. It would be really helpful if the purpose of this class was documented.